packages/cardano-services/test/cli.test.ts

```diff
@@ -1562,6 +1564,8 @@ describe('CLI', () => {
               callCliAndAssertExit(
                 {
                   args: [
+                    '--postgres-connection-string-asset',
+                    postgresConnectionStringAsset,
```

Comment:
`tx-submit` provider shouldn't depend on asset database, so this is probably redundant (same with the tx-submit test that uses env variables). It does depend on `handle` database because it's validating handle resolutions before submitting to the node

Reply:
addressed in this commit now: 80eb76d12e79aca67a1624cffdcc60efbf5ad819